hashring/hashring_test.go

```diff
@@ -52,42 +52,45 @@ func (d *dummyListener) HandleEvent(event events.Event) {
 	d.l.Unlock()
 }
 
-func TestEvents(t *testing.T) {
+func TestAddServer(t *testing.T) {
 	ring := New(farm.Fingerprint32, 10)
 	l := &dummyListener{}
 	ring.RegisterListener(l)
-	ring.emit(struct{}{})
-	assert.Equal(t, 1, l.events, "expected one event to be emitted")
-}
 
-func TestAddServer(t *testing.T) {
-	ring := New(farm.Fingerprint32, 10)
 	ring.AddServer("server1")
 	ring.AddServer("server2")
-
+	assert.Equal(t, 4, l.EventCount())
```

Comment:
I agree, but the mockEventListener is not yet in master right? We can merge the changes in there, or clean up when that lands.